.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+.vscode
+.idea
+.secrets
```

Comment:
I don't think that we put IDE-related directories in our `.gitignore` files in other repositories.

To fix this on your local machine, you can make a global `.gitignore` and add those there. Then Git will never complain about them in any repo. 😄

Reply:
removed from repo's `.gitignore`